extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -2691,22 +2708,22 @@ static public function setFlags( $city_id, $city_flags, $skip=false, $reason = '
 	static public function getFlags( $city_id ) {
 		global $wgWikicitiesReadOnly;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgWikicitiesReadOnly" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)